body.es6

```diff
@@ -20,4 +20,7 @@ const WorldInNumbersArticleBody = createVariant({
   Country,
   Industry,
 })(ArticleBodyTemplate);
-export { StandardArticleBody, WorldInNumbersArticleBody };
+const WorldIfArticleBody = createVariant(
+  innerArticleBodyComponents
+)(ArticleBodyTemplate);
+export { StandardArticleBody, WorldInNumbersArticleBody, WorldIfArticleBody };
```

Comment:
Note: Due to reusability we shouldn't use these kind of customs names for variables.
Anyway the name of the component itself make it not reusable so just a note.